libs/components/src/next/button/vl-button.css.ts

```diff
@@ -1,7 +1,5 @@
-import { mediaQuerySmall } from '@domg-wc/common-utilities/css/mixin/media-queries.mixin.css';
-import { focusOutline } from '@domg-wc/common-utilities/css/mixin/outline.mixin.css';
-import { waveAnimation } from '@domg-wc/common-utilities/css/mixin/animation.mixin.css';
-import { CSSResult, css, unsafeCSS } from 'lit';
+import { vlFocusOutlineMixin, vlMediaScreenSmall, vlWaveAnimationMixin } from '@domg-wc/common-utilities/css';
+import { css, CSSResult, unsafeCSS } from 'lit';
 
 const borderWidth = '0.2rem';
```

Comment:
Kan dit stuk niet vervangen worden door css-variabelen voor border-width & border-width-small & dan ook functie `differenceBetweenRegularBorderAndSmallBorder` vervangen door gwn een andere `calc()`?

```ts
const borderWidth = '0.2rem';
const borderWidthSmall = '0.1rem';
const differenceBetweenRegularBorderAndSmallBorder = `calc(${borderWidth} - ${borderWidthSmall})`;
```

Hetzelfde bij `vl-link.css.ts`

Reply:
Ticket aangemaakt, wordt opgenomen in https://www.milieuinfo.be/jira/browse/UIG-3153